README.md

```diff
@@ -1,8 +1,54 @@
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Dorjee Tenzin . Jarvis Consulting
 
+I have a Bachelor of Computing from the University of Guelph where I majored in Computer Science. Throughout my education, I gained a strong fundamental understanding of computer science concepts and paradigms which allow me to be adaptable in the face of new technologies.  I’m great at problem-solving and enjoy writing clean and robust code, more than ever I love learning new concepts, technologies, and methodologies and applying that knowledge and myself. This passion has led me to explore and familiarize myself with many modern application and web development frameworks. Although I only have formal experience as a Backend Engineer, I’m very comfortable with Frontend technologies. 
+
+In terms of mentality and outlook on workplace culture, I like to keep a healthy balance of passion and pragmatism. While I enjoy learning new things and writing beautiful, elegant code, I am mindful of time and business constraints within a larger collaborative environment. I have many experiences working and collaborating with a team to complete a project. During University I was a Teaching Assistance (TA) for The Analysis and Design of Computer Algorithms course in which I had to mentor Second Year students in collaboration with other TA’s and the professor. At my current occupation at Jarvis Consulting, I work in a team to complete projects using Agile and Scrum methodologies.  My strong academic background and my experience working and collaborating within a team will allow me to make meaningful contributions and be a great addition to your team.
+
+## Skills
+
+**Proficient:** C, Java, Python/Django, RDBMS/SQL/Postgres, Agile/Scrum, REST
+
+**Competent:** Javascript/React, NodeJS/Express, Docker, AWS/GCP, CSS
+
+**Familiar:** GraphQL, Springboot, TypeORM, TypeScript, Heroku
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_DorjeeTenzin/](https://github.com/jarviscanada/jarvis_data_eng_DorjeeTenzin/)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DorjeeTenzin//tree/master/linux_sql)]: Developed a MVP tool using bash monitoring agent to record hardware specifications and monitor resources usage for a Linux Cluster Administration team in real-time. The Cluster consists of a switch that connects multiple nodes/servers running CentOS 7 provisioned by Google Cloud Platform (GCP). The Cluster utilizes bash scripts to initialize a PostgresSQL Docker instance, create a schema, retrieve data from the nodes, and populate the database. Relevant data is isolated and filtered using SQL queries to generate meaningful reports. Version control is achieved through git for the local repository and GitHub for the remote repository, in addition, it adheres to Gitflow conventions.
```

Comment:
**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_DorjeeTenzin//tree/master/linux_sql)]: Developed an MVP tool using bash monitoring agent to record hardware specifications and monitor resources usage for a Linux Cluster Administration team in real-time. The Cluster consists of a switch that connects multiple nodes/servers running CentOS 7 provisioned by Google Cloud Platform (GCP). The Cluster utilizes bash scripts to initialize a PostgresSQL Docker instance, create a schema, retrieve data from the nodes, and populate the database. Relevant data is isolated and filtered using SQL queries to generate meaningful reports. Version control is achieved through git for the local repository and GitHub for the remote repository with Gitflow conventions.